Public/Src/App/Bxl/Args.cs

```diff
@@ -864,6 +864,10 @@ public bool TryParse(string[] args, PathTable pathTable, out ICommandLineConfigu
                         OptionHandlerFactory.CreateOption(
                             "tempDirectory",
                             opt => layoutConfiguration.TempDirectory = CommandLineUtilities.ParsePathOption(opt, pathTable)),
+                        OptionHandlerFactory.CreateBoolOption(
+                            "temporary_PreserveOutputsForIncrementalTool",

```

Comment:
>temporary_ [](start = 29, length = 10)

It should no longer be temporary.

Reply:
it is still temporary. we will get it removed when they pass processoptions.incrementaltool

---
In reply to: [310330852](https://github.com/microsoft/BuildXL/pull/697#discussion_r310330852) [](ancestors = 310330852)